htdocs/product/stock/product.php

```diff
@@ -725,178 +725,179 @@
 			$parameters = array();
 			$reshook = $hookmanager->executeHooks('formObjectOptions', $parameters, $object, $action); // Note that $action and $object may have been modified by hook
```

Comment:
The hook here is designed to add lines, inside the first columns.
My understanding is that you want to completely removed the second column ?
The pb with your PR is that it generates not correctly balanced open/close object. In one case the table is closed into a given level and into another one, it is closed into another one. We should avoid such cases as it makes html templates reading difficult. If goal is to hide completely the right column, why don't you add a simple CSS entry into the custom CSS provided by your module ?

Reply:
OK. 
I understand these remarks. 
I close the PR. Thanks for the time you spent to this issue.